components/brave_wallet_ui/components/buy-send-swap/buy/index.tsx

```diff
@@ -0,0 +1,69 @@
+import * as React from 'react'
+import {
+  AssetOptionType,
+  BuySendSwapViewTypes,
+  NetworkOptionsType,
+  ToOrFromType
+} from '../../../constants/types'
+import { NavButton } from '../../extension'
+import SwapInputComponent from '../swap-input-component'
+import locale from '../../../constants/locale'
+// Styled Components
+import {
+  StyledWrapper,
+  FaucetTitle,
+  FaucetWrapper,
+  FaucetDescription
+} from './style'
+
+export interface Props {
+  selectedAsset: AssetOptionType
+  selectedNetwork: NetworkOptionsType
+  buyAmount: string
+  onSubmit: () => void
+  onInputChange: (value: string, name: string) => void
+  onChangeBuyView: (view: BuySendSwapViewTypes, option?: ToOrFromType) => void
+}
+
+function Buy (props: Props) {
+  const {
+    selectedNetwork,
+    selectedAsset,
+    buyAmount,
+    onInputChange,
+    onSubmit,
+    onChangeBuyView
+  } = props
+
+  const onShowAssets = () => {
+    onChangeBuyView('assets', 'from')
+  }
+
+  return (
+    <StyledWrapper>
+      {selectedNetwork.id === 0 ? (
```

Comment:
It's better to use an enum here.

Reply:
https://github.com/brave/brave-core/pull/9575#discussion_r679496193